bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/BKException.java

```diff
@@ -240,4 +255,60 @@ public static String getMessage(int code) {
         int UnexpectedConditionException = -999;
     }
 
+    /**
+     * Code log message pool.
+     */
+    private static class LogMessagePool {
+        private static final int lastKnown = 200;
```

Comment:
Currently it seems OK. It would be better to have this connected with size of Code above, and not hard coded.

Reply:
@jiazhai sure, please see 7676dab